Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -163,6 +163,21 @@ directory :file:`_assets/`.
     :bash:`composer dumpautoload` and the :file:`Resources/Public/` folder for
     that extension is symlinked to :file:`_assets/`.
 
+..  warning::
+    The :file:`_assets/` directory is not meant be manually changed. Also, it
+    is important for local development that all subdirectories are symlinks
```

Comment:
"all subdirectories" -> you mean subdirectories under _assets?

Reply:
Also, here the "meant TO be" is missing.

Reply:
Yes, since `_assets` is established in the first sentence I think it is clear that subdirectories are relative to that term? I'd like to avoid duplicating this term too often, because it occurs very often on the rendered page :)

Reply:
If the duplication enhances clarity ...

Reply:
How about this:

```suggestion
    is important for local development that all its subdirectories are symlinks
```